web3.md

```diff
@@ -580,30 +580,75 @@ WEB3 JSON RPC
 **TODO**: You're supposed to be able to overwrite a pending tx by using its same "from" and "nonce" values
 
 ```k
-    syntax KItem ::= "loadTX" JSON
- // ------------------------------
-    rule <k> loadTX J => signTX !TXID ACCTADDR ... </k>
-         <txPending> .List => ListItem( !TXID ) ... </txPending>
+    syntax KItem ::= "createTX" Int JSON
+ // ------------------------------------
+    rule <k> createTX ACCTID J => loadTX !TXID:Int J ~> signTX !TXID:Int ACCTID ... </k>
+         <txPending> (.List => ListItem(!TXID)) ... </txPending>
          <account>
-           <acctID> ACCTADDR </acctID>
-           <nonce> ACCTNONCE => ACCTNONCE +Int 1 </nonce>
+           <acctID> ACCTID </acctID>
+           <nonce>  ACCTNONCE </nonce>
            ...
          </account>
          <messages>
-           (.Bag => <message>
-             <msgID> !TXID:Int </msgID>
-             <txNonce>    #if isString( #getJSON("nonce",    J) ) #then #parseHexWord(#getString("nonce",J))    #else ACCTNONCE  #fi </txNonce>
-             <to>         #if isString( #getJSON("to",       J) ) #then #parseHexWord(#getString("to",J))       #else 0          #fi </to>
-             <txGasLimit> #if isString( #getJSON("gas",      J) ) #then #parseHexWord(#getString("gas",J))      #else 90000      #fi </txGasLimit>
-             <txGasPrice> #if isString( #getJSON("gasPrice", J) ) #then #parseHexWord(#getString("gasPrice",J)) #else GASP       #fi </txGasPrice>
-             <value>      #if isString( #getJSON("value",    J) ) #then #parseHexWord(#getString("value",J))    #else 0          #fi </value>
-             <data>       #if isString( #getJSON("data",     J) ) #then #parseByteStack(#getString("data",J))   #else .ByteArray #fi </data>
-             ...
-           </message>)
-         ...
-         </messages>
-         <gasPrice> GASP </gasPrice>
-      requires #parseHexWord(#getString("from",J)) ==Int ACCTADDR
+            ( .Bag
+           => <message>
+                <msgID> !TXID </msgID>
+                <txGasPrice> 1         </txGasPrice>
+                <txNonce>    ACCTNONCE </txNonce>
+                <txGasLimit> 90000     </txGasLimit>
+                ...
+              </message>
+            )
+          ...
+          </messages>
+
+    syntax KItem ::= "loadTX" Int JSON
+ // ----------------------------------
+    rule <k> loadTX _ { .JSONList } => . ... </k>
+
+    rule <k> loadTX _ { ("from": _, REST) => REST } ... </k>
+
+    rule <k> loadTX TXID { ("to": TO_STRING, REST) => REST } ... </k>
```

Comment:
make these rules more useful by having them like this:

```
rule <k> loadTX _ { "to": (TO_STRING => #parseHexWord(TO_STRING)) , REST }
rule <k> loadTX TXID { "to": ACCOUNT_TO:Int , REST => REST } ... </k>
     <message>
       <msgID> TXID </msgID>
       <to> _ => ACCOUNT_TO </to>
       ...
     </message>
```

That way a given input can either have the hex-string directly, or the actual integer itself (for instance, if the integer is loaded from somewhere else in the configuration).